config/samples/mysql_v1alpha1_mysqlcluster_mysql8.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: sample
 spec:
   replicas: 3
-  mysqlVersion: "8.0"
```

Comment:
The same as above.

Reply:
MySQLVersion should be remained. and use this value to auto fill xenon, mysql , sidecar images. 
When the users fill these images, use the fill values to replace the auto filled values.

Reply:
We can use helm chart to achieve this. 
CR yaml I think should be keep full image name.
user can known which image he use through CR  and it would  be more flexible
@acekingke @runkecheng

Reply:
It must be 8.0